daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SValue.scala

```diff
@@ -151,25 +153,18 @@ object SValue {
 
   final case class STextMap(textMap: HashMap[String, SValue]) extends SValue
 
-  final case class SGenMap(genMap: InsertOrdMap[SGenMap.Key, SValue]) extends SValue
+  final case class SGenMap(genMap: TreeMap[SValue, SValue]) extends SValue
 
```

Comment:
Somewhere in here

```scala
  implicit def ` ordering`: Ordering[SValue] = svalue.Ordering
```

Then you should be able to remove all the explicit references.

Reply:
done